README.md

```diff
@@ -2,43 +2,45 @@
 
 ---
 
-**Table of Contents** &nbsp;&nbsp; [Cockatrice](#cockatrice) | [Downloads](#downloads-) | [Get Involved](#get-involved-) | [Community](#community-resources) | [Translation](#translation-status-) | [Building](#building--) | [Running](#running) | [License](#license-)
+**Table of Contents** &nbsp;&nbsp; [Cockatrice](#cockatrice) | [Downloads](#downloads-) | [Get Involved](#get-involved-) | [Community](#community-resources) | [Translation](#translation-status-) | [Building](#building--) | [Running](#running) | [ReadMe Style Guide](#readme-) | [License](#license-)
```

Comment:
>The style anchor is broken - I think it needs to be `#readme-`

Isn't it even `#readme-style-guide-and-documentation-process`? :)
Couldn't verify if it's working with that change on your fork, I can't access the changed README file on your branch you pr'ed from somehow.